test/testpy/__init__.py

```diff
@@ -53,11 +53,15 @@ def AfterRun(self, result):
     # delete the whole tmp dir
     try:
       rmtree(self.tmpdir)
+      if "NODE_PIPE_DIR" in os.environ:
+        rmtree(os.environ["NODE_PIPE_DIR"]);
```

Comment:
I'm concerned with what happens if users set `NODE_PIPE_DIR` to a directory that contains other data in it that is not temporary and that is not related to Node.js' tests.

Maybe we should always generate a temporary directory for pipes using something like `mktemp -d -t prefix`? I'm not sure how to do that on Windows though.